/motores:
  ros__parameters:
    FR:
      legType: "RIGHT"
      motor0:
        motorN: 0
        axisID: 0
        iOffset: -0.7781874978542329
        direction: -1
        calibration_angle: 35.6
        joint_limit_min: -30.0
        joint_limit_max: 30.0
      motor1:
        motorN: 0
        axisID: 1
        iOffset: 0.9250001764297486
        direction: -1
        calibration_angle: -71.9
        joint_limit_min: -60.0
        joint_limit_max: 45.0
      motor2:
        motorN: 1
        axisID: 2
        iOffset: -3.5598085719347
        direction: -1
        calibration_angle: 150.2
        joint_limit_min: 0.0
        joint_limit_max: 140.0
    FL:
      legType: "LEFT"
      motor0:
        motorN: 1
        axisID: 3
        iOffset: -0.05593751430511462
        direction: -1
        calibration_angle: -35.6
        joint_limit_min: -30.0
        joint_limit_max: 30.0
      motor1:
        motorN: 0
        axisID: 4
        iOffset: 1.7233085861802102
        direction: 1
        calibration_angle: -71.9
        joint_limit_min: -60.0
        joint_limit_max: 45.0
      motor2:
        motorN: 1
        axisID: 5
        iOffset: -3.635941390991211
        direction: 1
        calibration_angle: 150.2
        joint_limit_min: 0.0
        joint_limit_max: 140.0
    BL:
      legType: "LEFT"
      motor0:
        motorN: 0
        axisID: 6
        iOffset: -0.0540001845359801
        direction: 1
        calibration_angle: -35.6
        joint_limit_min: -30.0
        joint_limit_max: 30.0
      motor1:
        motorN: 0
        axisID: 7
        iOffset: 0.8501875996589661
        direction: 1
        calibration_angle: 0.0
        joint_limit_min: -90.0
        joint_limit_max: 20.0
      motor2:
        motorN: 1
        axisID: 8
        iOffset: -3.568941652774811
        direction: 1
        calibration_angle: 125.0
        joint_limit_min: 0.0
        joint_limit_max: 130.0
    BR:
      legType: "RIGHT"
      motor0:
        motorN: 1
        axisID: 9
        iOffset: -0.04669149160385144
        direction: 1
        calibration_angle: 35.6
        joint_limit_min: -30.0
        joint_limit_max: 30.0
      motor1:
        motorN: 0
        axisID: 10
        iOffset: 0.9440624713897705
        direction: -1
        calibration_angle: 0.0
        joint_limit_min: -90.0
        joint_limit_max: 20.0
      motor2:
        motorN: 1
        axisID: 11
        iOffset: -3.3376877456903458
        direction: -1
        calibration_angle: 125.0
        joint_limit_min: 0.0
        joint_limit_max: 130.0
